packages/frontend/src/utils/refreshStatusInfo.ts

```diff
@@ -0,0 +1,40 @@
+import { IconName } from '@blueprintjs/core';
+
+export const refreshStatusInfo = (
+    status: string,
+): { title: string; icon: IconName; status: string } => {
+    switch (status) {
+        case 'DONE':
```

Comment:
There is also a new state `STARTED` that I introduced to avoid you getting "no results" or 404 on the backend. But that should not make any difference here, just so you know